pylint/testutils/output_line.py

```diff
@@ -62,39 +76,42 @@ def __init__(self, row, exception):
 class OutputLine(NamedTuple):
     symbol: str
     lineno: int
-    column: str
-    object: Any
+    column: int
+    object: str
```

Comment:
I have checked the full test suite by printing object in `OutputLine.from_msg()`. This was always `str`, which is to be expected since we are dealing with `csv` and `output lines` here.